samples/snippets/src/main/java/com/example/bigtable/deletes/BatchDeleteExample.java

```diff
@@ -20,29 +20,24 @@
 import com.google.api.gax.batching.Batcher;
 import com.google.api.gax.rpc.ServerStream;
 import com.google.cloud.bigtable.data.v2.BigtableDataClient;
-import com.google.cloud.bigtable.data.v2.models.Filters;
 import com.google.cloud.bigtable.data.v2.models.Query;
 import com.google.cloud.bigtable.data.v2.models.Row;
 import com.google.cloud.bigtable.data.v2.models.RowMutationEntry;
 import java.io.IOException;
 
-public class DeleteStreamingAndBatchingExample {
-  public void streamingAndBatching(
-      String projectId, String instanceId, String tableId, String familyName, String qualifier) {
+public class BatchDeleteExample {
+  public void batchDelete(String projectId, String instanceId, String tableId)
+      throws InterruptedException, IOException {
     try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
```

Comment:
I don't think you need the try's for these since there is no catch, then no need for all the extra indentation etc.

Reply:
I was following: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#client-initialization. Should I undo this?